modules/tensor_mechanics/src/materials/ComputeIncrementalSmallStrain.C

```diff
@@ -79,3 +71,18 @@ ComputeIncrementalSmallStrain::computeProperties()
     _total_strain[_qp] = _total_strain_old[_qp] + total_strain_increment;
   }
 }
+
+void
+ComputeIncrementalSmallStrain::computeTotalStrainIncrement(RankTwoTensor & total_strain_increment, unsigned int qp)
+{
+  //Deformation gradient
+  RankTwoTensor A((*_grad_disp[0])[qp], (*_grad_disp[1])[qp], (*_grad_disp[2])[qp]); //Deformation gradient
+  RankTwoTensor Fbar((*_grad_disp_old[0])[qp], (*_grad_disp_old[1])[qp], (*_grad_disp_old[2])[qp]); //Old Deformation gradient
```

Comment:
Ok, this is exactly the _construct-from-rows_ syntax I suggested above. So you already know it. Let's use it consistently!